database/schema.sql

```diff
@@ -46,9 +44,11 @@ CREATE TABLE body_assignment (
     at TIMESTAMP NOT NULL,
     azimuth FLOAT NOT NULL,
     altitude FLOAT NOT NULL,
+    constellation_id SMALLINT,
     PRIMARY KEY (assignment_id),
     FOREIGN KEY (region_id) REFERENCES region(region_id),
-    FOREIGN KEY (body_id) REFERENCES body(body_id)
+    FOREIGN KEY (body_id) REFERENCES body(body_id),
```

Comment:
Nice changes to the ERD so we don't have to update the body table each time a body changes consetllation. Thanks for applying what was mentioned in the meeting with @Ruyzambrano  and @Peritract